ui/src/app/dashboard/dashboard.component.css

```diff
@@ -0,0 +1,23 @@
+* {
+  font-family: "Gotham HTF", sans-serif;
+}
+
+div {
+  margin: 1rem;
+}
+
+.aggregation-container {
+  padding: 1rem;
+}
+
+.group-container {
```

Comment:
I'm thinking justify-content: space-evenly probably makes sense here, rather than left justified

Reply:
Not addressed, I think.

Reply:
Still not addressed?

Reply:
Shouldn't this is what we just discussed? We are not going to change the `justify-content` to `space-evently`, right?

Reply:
We did, and I thought the outcome was that we would change this to `space-evenly` and the total summaries component to `space-between`. This should be unrelated to the overflowing columns issue.

Reply:
Sorry I misunderstood you. Fixed.